modules/cluster-logging-log-forward-update.adoc

```diff
@@ -0,0 +1,132 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-external.adoc
+
+// Might not be needed. See https://issues.redhat.com/browse/LOG-654
+
+[id="cluster-logging-log-forward-update_{context}"]
+= Updating log forwarding custom resources
+
+The {product-title} Log Forward API has been promoted from Technology Preview to Generally Available in {product-title} 4.6. The GA release contains some improvements and enhancements that require you to make a change to your Cluster Logging custom resource (CR) and to replace your `LogForwarding` custom resource (CR) with a `ClusterLogForwarder` CR.
+
+.Procedure 
+
+If you use the Log Forwarding API to forward logs, after the Elasticsearch Operator and Cluster Logging Operator are fully updated to 4.6, you must make the following modifications:
+
+. Edit the Cluster Logging custom resource (CR) to remove the `logforwardingtechpreview` annotation:
++
+.Sample Cluster Logging CR
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+metadata:
+  annotations:
+    clusterlogging.openshift.io/logforwardingtechpreview: enabled <1>
+  name: "instance"
+  namespace: "openshift-logging"
+....
+
+----
+<1> Remove the `logforwardingtechpreview` annotation.
+
+. Export the Log Forwarding CR to create a YAML file for the `ClusterLogForwarder` instance:
++
+[source,terminal]
+----
+$ oc get LogForwarding instance -n openshift-logging -o yaml| tee ClusterLogForwarder.yaml
+----
+
+. Edit the YAML file to make the following modifications:
++
+.Sample Log Forwarding CR in {product-title} 4.5
```

Comment:
I was momentarily confused at the back-to-back code blocks. I suggest swapping the order of the YAML examples and say something like "The modified YAML file resembles the following example:" before this one.